tagstudio/src/core/library.py

```diff
@@ -1352,10 +1352,13 @@ def search_library(
             only_missing: bool = "missing" in query or "no file" in query
             allow_adv: bool = "filename:" in query_words
             tag_only: bool = "tag_id:" in query_words
+            tag_only_ids: list[int] = []
             if allow_adv:
                 query_words.remove("filename:")
             if tag_only:
                 query_words.remove("tag_id:")
+                tag_only_ids.append(int(query_words[0]))
+                tag_only_ids = tag_only_ids + self.get_tag_cluster(int(query_words[0]))
```

Comment:
Will query_words[0] always be a digit? 
Use extend on tag_only_ids instead of adding two lists together.

Reply:
Was about to confidently say "Yes, the `tag_id:` prefix is only triggered automatically by the program" and then realized that if the user typed in something like "tag_id: gotcha" it would fail...